qemu/tests/virtio_fs_support_win_fs.py

```diff
@@ -30,60 +30,66 @@ def run(test, params, env):
     reg_add_pwd = params.get("reg_add_pwd")
     driver = params["driver_name"]
     driver_verifier = params.get("driver_verifier", driver)
-    driver_running = params.get('driver_running', driver_verifier)
+    driver_running = params.get("driver_running", driver_verifier)
 
     vm = env.get_vm(params.get("main_vm"))
     vm.verify_alive()
     session = vm.wait_for_login()
 
-    utils_test.qemu.windrv_verify_running(session, test,
-                                          driver_running)
-    session = utils_test.qemu.setup_win_driver_verifier(session,
-                                                        driver_verifier,
-                                                        vm)
+    utils_test.qemu.windrv_verify_running(session, test, driver_running)
+    session = utils_test.qemu.setup_win_driver_verifier(session, driver_verifier, vm)
     error_context.context("Create the viofs service.", test.log.info)
     virtio_fs_utils.create_viofs_service(test, params, session)
     error_context.context("Add NTFS filesystem to virtiofs.", test.log.info)
     reg_add_cmd = params.get("reg_add_cmd")
     session.cmd(reg_add_cmd)
-    error_context.context("restart virtiofs service to make the registry "
-                          "change work.", test.log.info)
+    error_context.context(
+        "restart virtiofs service to make the registry " "change work.", test.log.info
```

Comment:
"restart virtiofs service to make the registry " "change work." -> "restart virtiofs service to make the registry change work."

I guessed this was formatted by a 'tool'